kubernetes/istio/istio-api/src/main/java/org/arquillian/cube/istio/api/IstioResources.java

```diff
@@ -0,0 +1,14 @@
+package org.arquillian.cube.istio.api;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.TYPE, ElementType.ANNOTATION_TYPE})
+public @interface IstioResources {
```

Comment:
How about name it `List` and move it to be inside the IstioResource annotation (just like what javax.validation API does)?

Reply:
I'll take a look, but then I will change also the other places where we are using it like `@OpenShiftResources` or `@Templates` to use the same approach in all annotations.

Reply:
Love the pattern, can I blog about it? I'll say that the idea comes from `javax.validation` code of course hehehe